scripts/xslt/cdrh_to_html/lib/cdrh.xsl

```diff
@@ -122,8 +122,8 @@
           </xsl:choose>
         </xsl:variable>
         <xsl:choose>
-          <xsl:when test="contains($figure_id_full,'.jpg')">
-            <xsl:value-of select="substring-before($figure_id_full,'.jpg')"/>
+          <xsl:when test="ends-with($figure_id_full,'.jpg') or ends-with($figure_id_full,'.jpeg')">
+            <xsl:value-of select="substring-before($figure_id_full,'.jp')"/>
```

Comment:
I hope Laura never names some project with `.jp` in the name of we're gonna have trouble :)